### DISCUSSION TITLE
How to sync a vault from PC to mobile

### DISCUSSION THREAD

**Discussion by GalacticViper** [remotely-save/remotely-save#382] [Q&A]
I have 2 vaults in question. I will call them my "phone vault" and my "PC vault" The PC vault was created on my PC and the phone one on my phone through the Obsidian Mobile app. (I am running Android 11)
When I installed remotely save on my PC vault and authenticated it to OneDrive, I could not open the vault on my phone. I tried going to OneDrive on my phone and set the whole remotely-save folder to be available offline, but Obsidian cannot see the folder no matter what I do.
However, I went into my phone vault and installed remotely-save on that, authenticated it to OneDrive, and I am able to access that one and make changes from my PC and visa versa perfectly.
but I still cant get my PC vault to be on my phone.
I think this is because when you mark files or folders “offline”, they are stored in a cache and the actual location of this cache is hidden and cannot be accessed directly.

I'm thinking I could just create a new vault on my phone sync it like I did with my phone vault, and then copy and paste everything to it, but Im kind of curious if anyone has gotten it to work from PC to mobile for them and how. 
The plugin page for remotely save says you can use the cloud service as a "broker", but I'm not fully sure how I would accomplish that with OneDrive.

**Comment by giggio** [comment]
I agree this is not well documented. Or at all...
I have made it work by reading other posts in the issues/dicussions. You need to create a Vault with the same name, install the plugin manually, reauth, and it will sync as expected.
And it seems that the plugin settings do not sync, even you choose to select the Obsidian settings.

**Reply by cartilage-ftw** [reply]
Thank you, your answer helped! 

After doing this, though, I'm still only able to synchronize one vault at a time. Is it possible to synchronize multiple vaults and be able to browse them? For instance I might have three vaults 'notes-general', 'atomic', 'nuclear' on my laptop. I want all three separate vaults to be synchronized via dropbox, and there be an option to choose which one I want to view/edit on the other device.

**Reply by giggio** [reply]
I don't know, I haven't tried it.

**Reply by pkfire** [reply]
Hi Giggio, 
I was haveing the same problem as OP and I used your suggestions. It worked on the phone!
What settings do you use to sync on your PC and your phone and why did you choose those? 
Thank you for your help as it is not well documented

**Reply by giggio** [reply]
@pkfire I didn't change most of the options.

**Reply by sucosama** [reply]
> I agree this is not well documented. Or at all... I have made it work by reading other posts in the issues/dicussions. You need to create a Vault with the same name, install the plugin manually, reauth, and it will sync as expected. And it seems that the plugin settings do not sync, even you choose to select the Obsidian settings.

thank you so much! this helped my case. I was almost drop this altogether and apply for an education version just for sync function

**Reply by Damaskox** [reply]
> I agree this is not well documented. Or at all... I have made it work by reading other posts in the issues/dicussions. You need to create a Vault with the same name, install the plugin manually, reauth, and it will sync as expected. And it seems that the plugin settings do not sync, even you choose to select the Obsidian settings.

I just hopped into this plugin. I also had a separate vault name on my mobile compared to PC. Made a new vault on mobile with the same name than on PC. Enabled plugins on both machines, logged in to Dropbox with both and enabled both "Run once on start up automatically" (sync once after 1 second of startup) and "sync on save (experimental)" Closed app on mobile, reopened and watched how it all downloaded, resulting in the same amount of files and folders than what I have on my PC 🥳 Then disabled the "sync on startup" and left the "Sync on save" on

Let us see whether I get problems on a longer run!
